omod/src/main/web/dashboardwidgets/datepicker/datepicker.html

```diff
@@ -1,5 +1,6 @@
 <div class="input-group date">
-    <input type="text" class="form-control" readonly="readonly">
+    <!-- TODO: make form-control-sm configurable -->
```

Comment:
What do you mean by this? You shouldn't override bootstrap classes. Just add additional classes when needed.

Reply:
I just meant that users of the openmrs-datepicker widget should be able to choose between the bootstrap normal and bootstrap small layout (ie whether or not the form-control-sm class is applied here).  I'll update the comment, to (hopefully!) make it clearer.